zarr/zarr-in-practice.ipynb

```diff
@@ -8,11 +8,38 @@
     "\n",
```

Comment:
<p>Is there a line break "I\nn order to limit requests..." or is that just how it's appearing in the review...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>